ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -627,15 +628,15 @@ int mca_pml_ucx_recv(void *buf, size_t count, ompi_datatype_t *datatype, int src
     MCA_COMMON_UCX_PROGRESS_LOOP(ompi_pml_ucx.ucp_worker) {
         status = ucp_request_test(req, &info);
         if (status != UCS_INPROGRESS) {
-            mca_pml_ucx_set_recv_status_safe(mpi_status, status, &info);
+            result = mca_pml_ucx_set_recv_status_safe(mpi_status, status, &info);
```

Comment:
do we need this also in other places mca_pml_ucx_set_recv_status_safe is used?

Reply:
fixed for MPI_Mrecv call, other locations are non-blocking MPI calls - not relevant